easybuild/easyblocks/c/cuda.py

```diff
@@ -136,8 +137,24 @@ def create_wrapper(wrapper_name, wrapper_comp):
         for comp in (self.cfg['host_compilers'] or []):
             create_wrapper('nvcc_%s' % comp, comp)
 
+        saved_path = os.environ.get('PATH', '')
+        ldconfig = which('ldconfig')
+        if ldconfig:
+            self.log.info("ldconfig found at %s", ldconfig)
+        else:
+            # ldconfig is usually in /sbin or /usr/sbin
+            for cand_path in ['/sbin', '/usr/sbin']:
+                if os.path.exists(os.path.join(cand_path, 'ldconfig')):
+                    self.log.info("ldconfig found at %s, so appending that location to $PATH" % cand_path)
+                    setvar('PATH', os.pathsep.join([saved_path, cand_path]))
```

Comment:
@akesandgren I think it makes sense to avoid updating `$PATH` all together... We can just assign `ldconfig` to the full path to `ldconfig`, and then use the `ldconfig` variable in `run_cmd`, so we (always) call `ldconfig` full path?

That's safer than pushing `/sbin` into `$PATH` (and less code since we don't need to set/restore `$PATH`)